src/components/tx/ErrorMessage/index.tsx

```diff
@@ -14,7 +15,7 @@ const ErrorMessage = ({
   children: ReactNode
   error?: Error & { reason?: string }
   className?: string
-  level?: 'error' | 'warning' | 'info'
+  level?: 'error' | 'warning' | 'info' | 'border'
```

Comment:
Should we use a different word? I know it is the border color but this mixes domains quite a bit.
Should we call it "neutral" instead?